qcfractal/storage_sockets/models/sql_models.py

```diff
@@ -247,12 +248,12 @@ def base_result(self, val):
     # explicit sort will have to process all the data to identify the first n
     # rows, but if there is an index matching the ORDER BY, the first n rows
     # can be retrieved directly, without scanning the remainder at all.
-
     __table_args__ = (
         Index("ix_task_queue_created_on", "created_on"),
         Index("ix_task_queue_keys", "status", "program", "procedure", "tag"),
         Index("ix_task_queue_manager", "manager"),
         Index("ix_task_queue_base_result_id", "base_result_id"),
+        Index("ix_task_waiting_sort", text("priority desc,  created_on")),
```

Comment:
Why did you need the `text` here?

Reply:
https://docs.sqlalchemy.org/en/13/core/constraints.html#functional-indexes
Functional indexes as properties associated with the class table are only supported through the ```text()``` construct